File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -299,6 +299,80 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-first-occurance).
 
+
+## Find and replace using Regex pattern 
+
+You can find all occurrences of a pattern of text in a PowerPoint presentation using the FindAll method and replace it with other text.
```

Comment:
@Venkateshmuruganandam - Can you please explain about `Find` API in this sentence?